parquet-column/src/main/java/org/apache/parquet/filter2/predicate/Operators.java

```diff
@@ -315,6 +321,66 @@ public <R> R accept(Visitor<R> visitor) {
     }
   }
 
+  public abstract static class ArrayContainsFilterPredicate<T extends Comparable<T>>
```

Comment:
It seems this class does not need to be public. Let's have it package private.

Reply:
sounds good! I think I'll have to refactor this API design a bit to make Array filters composable, as @wgtmac suggested above -- will try to push changes late today or Monday 🙏